eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -121,25 +148,25 @@ jobs:
       maxParallel: $[ variables['MaxParallelTestJobs'] ]
       matrix:
         Linux:
-          Pool: # Intentionally blank.
-          OSVmImage: "ubuntu-18.04"
+          Pool: ${{ parameters.LinuxPool }} # Comment out to swap back to public hosted pool.
+          OSVmImage: # "ubuntu-18.04" # Comment back in to swap back to public hosted pool.
           TestTargetFramework: netcoreapp2.1
         Windows_NetCoreApp:
-          Pool: azsdk-pool-mms-win-2019-general # Comment out to swap back to public hosted pool.
+          Pool: ${{ parameters.WindowsPool }} # Comment out to swap back to public hosted pool.
           OSVmImage: # "windows-2019" # Comment back in to swap back to public hosted pool.
           TestTargetFramework: netcoreapp2.1
           CollectCoverage: true
         Windows_NetCoreApp_ProjectReferences:
-          Pool: azsdk-pool-mms-win-2019-general # Comment out to swap back to public hosted pool.
+          Pool: ${{ parameters.WindowsPool }} # Comment out to swap back to public hosted pool.
           OSVmImage: # "windows-2019" # Comment back in to swap back to public hosted pool.
```

Comment:
At what point do we think we should remove these empty properties? As for my other comment about could we control these at the higher level I guess we could have 2 parameters that we treat as mutually exclusive WindowsPool and WindowsImage and we can just have one be empty and the other set in order to toggle between these 2 if we need that functionality.

Reply:
Also we need to figure out how we want to handle the verify-agent-os templates when we start to run with a larger matrix. Personally I think it is even more important to ensure we are running the expected agent now that we are going to be swapping around pools.